app/controllers/application_controller.rb

```diff
@@ -13,19 +13,11 @@ class ApplicationController < ActionController::Base
 
   # :nocov:
 
-  def append_to_log(params)
-    @custom_log_items ||= {}
-    @custom_log_items.merge!(params)
-  end
-
   # Looks rather strange, but this is the suggested mechanism to add extra data
   # into the event passed to lograge's custom options. The method is part of
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
```

Comment:
@mloughran Yep.  As per the conversation we had at the time I merged it--I refactored to take your comments on that PR into account.  @alan  and I discussed it and it seemed like the most sensible approach.